plugins/org.eclipse.n4js/src/org/eclipse/n4js/parser/InternalSemicolonInjectingParser.java

```diff
@@ -44,22 +40,17 @@
  */
 public class InternalSemicolonInjectingParser extends InternalN4JSParser implements SemicolonInjectionHelper.Callback {
 
-	private static final Field ALL_RULES_FIELD;
```

Comment:
Why could you remove the code from this class?

Reply:
The original bug in Xtext was fixed in the meantime. The rules are now properly cached.